src/components/subscriptions/SubscriptionManagementPage.jsx

```diff
@@ -86,11 +86,11 @@ function SubscriptionManagementPage({ enterpriseSlug, enterpriseId }) {
                         </div>
                         <div className="col-12 col-lg-7">
                           <AddUsersButton
-                            onSuccess={() => {
+                            onSuccess={(response) => {
```

Comment:
Can we camelCase the `response` so we can access the keys without snake case (camelCase is convention in JS), e.g. `num_already_associated` --> `numAlreadyAssociated`?

I'd recommend doing that as soon as we get the data back from the API: https://github.com/edx/frontend-app-admin-portal/blob/master/src/components/AddUsersModal/index.jsx#L113

Reply:
We could also only pass `response.data` into the `onSuccess` function as well. `.data` is a thing that comes with axios requests (the way we make API calls).

Example:
```
const result = camelCaseObject(response.data);
this.props.onSuccess(result);
```

See https://github.com/edx/frontend-app-admin-portal/blob/53c218b68f3d63360973e5df2ad110d0fe7ccb4d/src/utils.js#L123 for where `camelCaseObject` is defined.